src/vconsole/vconsole-setup.c

```diff
@@ -271,20 +253,35 @@ static void setup_remaining_vcs(int fd, int vc_max, bool utf8) {
                         cfo.op = KD_FONT_OP_SET;
         }
         if (r < 0)
-                (void) log_warning_errno(errno, "%s failed, fonts will not be copied: %m", err_ioctl);
+                log_warning_errno(errno, "%s failed, fonts will not be copied: %m", err_ioctl);
 
         for (i = 1; i <= vc_max; i++) {
-                fd_d = -1;
-                if (i == vcs.v_active || !is_allocated(fd_d, i) || !is_settable(&fd_d, i))
+                /* note: cleanup function /is/ called per loop iteration, this is not a bug */
```

Comment:
Please remove this comment... Cleanup functions are used pretty much everywhere in systemd, in too many places to have comments about usage.